### ISSUE TITLE
NPM modules via URL

### ISSUE THREAD

**Issue by SevereOverfl0w** [eclipsesource/J2V8#187]
I'm trying to use some NPM modules in my code. I want to use webjars as the source of those modules, so that users of the software do not need to have `npm` installed.

Is it possible to refer to modules on the classpath in some way for `nodejs.require()`?

**Comment by midzelis** [comment]
Yes - I was able to do this. I needed to redefine fs.readFile* an fs.exists*. That almost did it, but node.js's module.js loader doesn't actually use fs.exists* for performance reasons. I couldn't redefine the native stat() function it used, so I had to go one layer up. I redefined     module._findPath, which needed to bring with it a pretty significant portion of the module loader code unfortunately. 

And it works! Needless to say, this is highly implementation dependent. As node changes, this could/will break. 

A nice feature request would be have J2V8 expose all file operations (native and not) as java callbacks.
